client/cgo/cgotest.go

```diff
@@ -0,0 +1,235 @@
+package main
+
+/*
+int testVRFVerify(unsigned char *pk, int pkSize,
+    unsigned char *m, int mSize,
+    unsigned char *index, int indexSize,
+    unsigned char *proof, int proofSize) {
+    if (cgoVrfVerify(pk, pkSize, m, mSize,
```

Comment:
I just want to make sure the flow: converting from golang types to C types and then from C types to golang types works as expected.